packages/formal/__tests__/use-formal.test.ts

```
@@ -218,7 +225,47 @@ describe('useFormal()', () => {
 
     it('should call clearErrors() before validating', async () => {})
     it('should change isValidating only if schema contains async validations', () => {})
-    it('should call setErrors() if the validation failed', () => {})
+    it('should set the errors in state if the validation failed', async () => {
```

I wonder if it is better to have the same description as before (`should call setErrors() if the validation failed`). And within the `catch`, check if `formal.setErrors` have been called by using [jest spies](https://jestjs.io/docs/en/jest-object.html#jestspyonobject-methodname).

Also, it makes more sense to put first the `should throw the errors if the validation failed` test and then this one.